wca-regulations.md

```diff
@@ -419,7 +419,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - A4d3) If a stopwatch is in use, the judge starts the stopwatch as soon as the competitor starts the solve.
     - A4e) Time penalties for starting the solve are cumulative.
 - A5) During the solve:
-    - A5a) While inspecting or solving the puzzle, the competitor must not communicate with anyone other than the judge or the WCA Delegate. Penalty: disqualification of the attempt (DNF).
+    - A5a) While inspecting or solving the puzzle, the competitor should not communicate with anyone other than the judge or the WCA Delegate. Penalty: disqualification of the attempt (DNF).
```

Comment:
```suggestion
    - A5a) While inspecting or solving the puzzle, the competitor must not communicate with anyone other than the judge or the WCA Delegate. Penalty: disqualification of the attempt (DNF). Exception: If communication does not give an unfair advantage, the result may stand, at the discretion of the WCA Delegate. 
```

Other possible wordings:

- Exception: If this conversation was not initiated by the competitor, the result may stand, at the discretion of the WCA Delegate.